Gratwurst.lua

```diff
@@ -202,34 +154,63 @@ function OnEventReceived(self, event, msg, author, ...)
 	end
 end
 
-function GuildAchievementMessageEventReceived()
-	gratsStop=true
+function GuildAchievementMessageEventReceived(isDebug, author)
+	local gratsStop = true
+	local canGrats = false
+	if GratwurstShouldVary then
+		local random = math.random(1, 100)
+		if random <= GratwurstVariancePercentage then
+			canGrats = true
+		end
+	end
```

Comment:
```suggestion
	if GratwurstVariancePercentage == 100 then
        canGrats = true
    else
		local random = math.random(1, 100)
		if random <= GratwurstVariancePercentage then
			canGrats = true
		end
	end
```

Reply:
This fixes the issues with GratwurstShouldVary always being false.   I don't think it's a necessary variable

Reply:
Oh I think initially I had a checkbox to enable it. Good catch.